app/components/NetStats/BigChartsSectionItem/TopLeft/index.js

```diff
@@ -7,6 +7,11 @@ import Value from '../Value';
 
 
 class TopLeft extends React.Component {
+  handleClickAction(data) {
```

Comment:
Can't you wrap it in an anchor element? `window.open` might hit the popup blockers :)